pom.xml

```diff
@@ -37,6 +37,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+        <dependency>
```

Comment:
We can improve client's calls in order to be a fault tolerant services with spring-retry for 5XX http status response errors